src/main/java/com/calendarfiles/Gcal.java

```diff
@@ -110,16 +110,22 @@ public static void printUpcomingEvents(Calendar cal, DateTime now) throws IOExce
         }
     }
 
-    public static void getFreeBusy(Calendar cal, DateTime startTime, DateTime endTime) throws IOException, GeneralSecurityException {
+    public static String getFreeBusy(Calendar cal, DateTime startTime, DateTime endTime) {
```

Comment:
Changes look great! Would recommend adding a header comment describing briefly what this function does.